packages/elvis-dropdown-lit/src/e-dropdown.ts

```diff
@@ -0,0 +1,169 @@
+import arrowDownBold from '@elvia/elvis-assets-icons/dist/icons/arrowDownBold';
+import { getShadow, getThemeColor } from '@elvia/elvis-colors';
+import { provide } from '@lit/context';
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import { DropdownContext, dropdownContext } from './dropdown-context';
+
+const arrowDownIcon = arrowDownBold.getIcon();
+
+@customElement('e-dropdown')
+export class EDropdown extends LitElement {
+  static styles = css`
+    :host {
+      --width: 448px;
+      --option-height: 48px;
+      display: block;
+      width: var(--width);
+      position: relative;
+    }
+
+    .input {
+      width: inherit;
+      border: 1px solid ${unsafeCSS(getThemeColor('border-1'))};
```

Comment:
We could consider removing the need for the call to `unsafeCss` by using the CSS variables directly, in other words `var(--e-color-border-1)`.

Reply:
While I kind of agree, I also think it's nice that this includes the fallback color-value, in case the component is used
1. without @elvia/elvis
2. with an unsupported version of @elvia/elvis, since there is no explicit version required in the `package.json`